lib/event_bus/services/store.ex

```diff
@@ -37,8 +39,13 @@ defmodule EventBus.Service.Store do
   @spec fetch(event_shadow()) :: event() | nil
   def fetch({topic, id}) do
     case Ets.lookup(table_name(topic), id) do
-      [{_, %Event{} = event}] -> event
-      _ -> nil
+      [{_, data}] -> data
+      _ ->
+        Logger.log(:info, fn ->
+          "[EVENTBUS-STORE]\s#{topic}.#{id}:\s#{:ets_error}"
```

Comment:
Maybe just use the same tag for the `[EVENTBUS]` on both places and add another second tag like `[EVENTBUS][STORE]\s#{topic}.#{id}.ets_fetch_error"`

Reply:
Okay,  so its `[EVENTBUS][STORE]` and `[EVENTBUS][OBSERVATION]` for `store.ex` and `observation.ex`